NKL/PrettyPrint.lean

```diff
@@ -0,0 +1,68 @@
+/-
+Copyright (c) 2024 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Paul Govereau
+-/
+
+import NKL.NKI
+
+namespace NKL
+
+instance : ToString Const where
```

Comment:
There's a nice wadler-style [pretty printer library](https://leanprover-community.github.io/mathlib4_docs/Init/Data/Format/Basic.html) that may be nice longer term.

Reply:
Ah, yes. I did see that. I will switch over to this. Thanks.